Lib/importlib/abc.py

```diff
@@ -97,7 +106,13 @@ def find_loader(self, fullname):
         This method is deprecated in favor of finder.find_spec(). If find_spec()
         is provided than backwards-compatible functionality is provided.
 
+        Deprecated since Python 3.4 in favor of find_spec (available since 3.4)
```

Comment:
This line is somewhat redundant since the deprecation is mentioned in the previous paragraph. Maybe just add the version it was deprecated in above?